lib/index.js

```diff
@@ -0,0 +1,8 @@
+const helpers = require('./helpers')
+
+/**
+ * Shared library
+ */
+module.exports = {
+  helpers
+}
```

Comment:
Yep totally agree, there's a later commit that separated "helpers" from "lib" so we could flatten this

I'm happy with destructuring though as it's similar to `const { dirname, join } = require('path')`

But if we want to avoid it we have some options:

### Namespace-style objects
```js
import { files } from 'govuk-frontend-lib'

files.getComponentsData()   // versus getComponentsData()
files.getDirectories()      // versus getDirectories()
files.getFullPageExamples() // versus getFullPageExamples()
```

### Named package exports
```js
import { getComponentsData, getDirectories, getFullPageExamples } from 'govuk-frontend-lib/files'

getComponentsData()
getDirectories()
getFullPageExamples()
```

Let me know

Reply:
Replied on the main thread of comments 😄